modules/vectra.py

```diff
@@ -2475,3 +2494,948 @@ def delete_account_note(self, account_id=None, note_id=None):
 
         return requests.delete('{url}/accounts/{account_id}/notes/{note_id}'.format(url=self.url, account_id=account_id, note_id=note_id), 
             headers=self.headers, verify=self.verify)
+
+
+class VectraClientV3(VectraClientV2_2):
+
+    def __init__(self, url=None, client_id=None, client_secret=None, verify=False):
+        """
+        Initialize Vectra client
+        :param url: IP or hostname of Vectra brain (ex https://www.example.com) - required
+        :param token: API token for authentication when using API v2*
+        :param verify: Verify SSL (default: False) - optional
+        """
+        # Remove potential trailing slash
+        url = VectraClientV3._remove_trailing_slashes(url)
```

Comment:
_remove_trailing_slashes() and _get_oauth_token() are not class methods.  You should be able to access them via self.

Reply:
Done